pkg/network/ebpf/c/protocols/classification/protocol-classification.h

```diff
@@ -149,7 +152,7 @@ __maybe_unused static __always_inline void protocol_classifier_entrypoint(struct
 
     protocol_stack_t *protocol_stack = get_protocol_stack_if_exists(&usm_ctx->tuple);
 
-    if (is_fully_classified(protocol_stack) || is_protocol_layer_known(protocol_stack, LAYER_ENCRYPTION)) {
```

Comment:
The change is not good. Now HTTPs (or gRPC tls, or kafka TLS) will be processed by the socket filter while it should not

Reply:
I think that traffic would only get a few lines further down before being returned by this block
```
        if (tls_hdr.content_type == TLS_APPLICATION_DATA) {
            // We can't classify TLS encrypted traffic further, so return early
            return;
        }
```
admittedly this is not perfect, but we won't proceed with the more complex tail calls

Reply:
ok I should be able to revert this